lib/shinseibank/cli/account.rb

```diff
@@ -0,0 +1,36 @@
+require "ostruct"
+require "shinseibank/cli/subcommand"
+
+class ShinseiBank
+  class CLI < Thor
+    class Account < Subcommand
+      desc "show", "Show your account details"
+      def show
+        login
```

Comment:
@knshiro  Didn't find anything nice... There's a Thor Hollaback gem, to attach before/after/around callbacks, but that feels a bit too much.